### PR TITLE
added success and error alerts for user authentication and login/signup

### PR DESCRIPTION
**Pull request by KimLinhVu**
What
===
* Add success and error alerts for ...
* Something else

Why
===
* Give the user better feedback for errors.

Who
===
@makushline @qibolun 

Test Plan
===
[](screenshot or video)

### CODE REVIEW

File: client/src/components/App/App.jsx [hunk 1]
```diff
@@ -9,11 +8,36 @@ import PlaylistDetail from '../PlaylistDetail/PlaylistDetail'
...
 import RecommendView from '../RecommendView/RecommendView'
 import UserPlaylistDetail from '../UserPlaylistDetail/UserPlaylistDetail'
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
 
 function App() {
-  const [signupMessage, setSignupMessage] = useState('')
   const {token, setToken, clearToken} = useToken()
 
+  const notifySuccess = (successMessage) => {
```
Comment (makushline) [inline client/src/components/App/App.jsx]: These notifications could be in their own file. One thing I recommend is keeping your App.jsx as light as possible. This way your app is a 'composition' of libraries..

File: server/app.js [hunk 2]
```diff
@@ -90,11 +89,9 @@ app.post('/verify-token', (req, res) => {
   const { token } = req.body
   jwt.verify(token, process.env.ACCESS_TOKEN_SECRET, (err) => {
     if (err) {
-      // res.json({ auth: false })
-      res.send(401, 'Failed to Authenticate')
+      res.status(401).send('Failed to authenticate')
```
Comment (makushline) [inline server/app.js:92]: 🙌🏾
Awesome work changing to using status codes. This is the more traditional way of communicating http responses.

**Review by makushline — APPROVED** [review]
Question, how come you have a lot of passing around the notify functions? 🤔
```
notifyError={notifyError}
notifySuccess={notifySuccess}
```
This may just be my own rudimentary understanding of React but can you help me understand that a bit more.

File: client/src/components/App/App.jsx (hunk 1 above)
Comment (KimLinhVu) [reply client/src/components/App/App.jsx]: Oops, no you are right, I forgot that putting them in one file was an option...

**Pull request merged**